tools/rpkg/R/dbDisconnect__duckdb_connection.R

```diff
@@ -6,7 +6,7 @@
 #' @param shutdown Set to `TRUE` to shut down the DuckDB database instance that this connection refers to.
 #' @rdname duckdb
 #' @usage NULL
-dbDisconnect__duckdb_connection <- function(conn, ..., shutdown = FALSE) {
+dbDisconnect__duckdb_connection <- function(conn, ..., shutdown = TRUE) {
```

Comment:
The ` duckdb_shutdown(conn@driver)` call in this function will call `dbDisconnect__duckdb_connection` creating the potential for infinite recursion.

Reply:
I think I noticed some weird behavior with this branch. I'm thinking maybe the recursive call should be avoided.